Add Code of Conduct (#80)
shalb/cluster.dev: CODE_OF_CONDUCT.md

# Community Code of Conduct

Cluster.dev follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).

<!-- markdownlint-disable no-inline-html -->

> Instances of abusive, harassing, or otherwise unacceptable behavior in Cluster.dev may be reported by contacting the SHALB via <SOME_EMAIL@shalb.com>

Reviewer: ```suggestion
Instances of abusive, harassing, or otherwise unacceptable behaviour in Cluster.dev should be reported by contacting SHALB via <SOME_EMAIL@shalb.com>
```
